__tests__/index.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from "@testing-library/react";
```

Comment:
Yeah, that's possible too. It's actually related to why I mentioned about the components in separate folders and keeping the pages-folder clean - at some point (within a year), there was some problems having the tests in `pages`-folder, because build didn't pass or something similar. Can't remember the details anymore 😅  But if we keep the "root"-components in `components`-folder, and test them there, and then just import them to the files `pages`-folder and pass data to them there, this should be ok from testing point of view as well. 

And if we want to use something like Cypress for E2E-testing, it anyway has the test-files in it's own folder, so this won't be a problem with them.

From this PR's perspective; I'd probably leave the tests to the `__tests__`-folder for now, and then move them when we have the `components`-folder in place. Would that be ok?

Reply:
Totally ok 👍 

I usually use Nightwatch for e2e-tests, but no harm in studying Cypress as well 😊